common/math/math.go

```diff
@@ -34,3 +46,195 @@ func RoundFloat(x float64, prec int) float64 {
 	pow := math.Pow(10, float64(prec))
 	return math.Round(x*pow) / pow
 }
+
+// CompoundAnnualGrowthRate Calculates CAGR.
+// Using years, intervals per year would be 1 and number of intervals would be the number of years
+// Using days, intervals per year would be 365 and number of intervals would be the number of days
+func CompoundAnnualGrowthRate(openValue, closeValue, intervalsPerYear, numberOfIntervals float64) (float64, error) {
+	if numberOfIntervals == 0 {
+		return 0, errCAGRNoIntervals
+	}
+	if openValue == 0 {
+		return 0, errCAGRZeroOpenValue
+	}
+	k := math.Pow(closeValue/openValue, intervalsPerYear/numberOfIntervals) - 1
+	return k * 100, nil
+}
+
+// CalmarRatio is a function of the average compounded annual rate of return versus its maximum drawdown.
+// The higher the Calmar ratio, the better it performed on a risk-adjusted basis during the given time frame, which is mostly commonly set at 36 months
+func CalmarRatio(highestPrice, lowestPrice, average float64) (float64, error) {
```

Comment:
Average value calculation isn't current accounting for the risk free rate

Reply:
I see conflicting sources that says that Calmar = 
`Average Annual Rate of Return / Maximum Drawdown`
or
`Rp – Rf / Maximum Drawdown`
or
`CAGR/ Maximum Drawdown`

I'm going to need sources before I change this

Reply:
https://corporatefinanceinstitute.com/resources/knowledge/trading-investing/calmar-ratio/#:~:text=The%20Calmar%20ratio%20is%20determined,it%20by%20its%20maximum%20drawdown.&text=The%20annual%20rate%20of%20return,been%20doing%20over%20a%20year.

Reply:
Further research by both of us seems to lean more towards including risk free rate. So yeah I'll include it